packages/belos/tpetra/example/BlockGmres/BlockGmresTpetraExFile.cpp

```diff
@@ -0,0 +1,261 @@
+    //
+    ParameterList belosList;
+    belosList.set( "Num Blocks", maxsubspace);             // Maximum number of blocks in Krylov factorization
+    belosList.set( "Block Size", blocksize );              // Blocksize to be used by iterative solver
+    belosList.set( "Maximum Iterations", maxiters );       // Maximum number of iterations allowed
+    belosList.set( "Maximum Restarts", maxrestarts );      // Maximum number of restarts allowed
+    belosList.set( "Convergence Tolerance", tol );         // Relative convergence tolerance requested
+    belosList.set( "Orthogonalization", ortho );           // Orthogonalization used by iterative solver
+    int verbosity = Belos::Errors + Belos::Warnings;
+    if (verbose) {
+      verbosity += Belos::TimingDetails + Belos::StatusTestDetails;
+      if (frequency > 0)
+        belosList.set( "Output Frequency", frequency );
+    }
+    if (debug) {
+      verbosity += Belos::Debug;
+    }
+    belosList.set( "Verbosity", verbosity );
+    //
+    // Construct an unpreconditioned linear problem instance.
+    //
+    Belos::LinearProblem<ST,MV,OP> problem( A, X, B );
+    bool set = problem.setProblem();
+    if (set == false) {
+      if (proc_verbose)
+        std::cout << std::endl << "ERROR:  Belos::LinearProblem failed to set up correctly!" << std::endl;
+      return -1;
+    }
+    //
+    // *******************************************************************
+    // *************Start the block Gmres iteration*************************
+    // *******************************************************************
+    //
+    // Create an iterative solver manager.
+    RCP< Belos::SolverManager<ST,MV,OP> > newSolver
+      = rcp( new Belos::BlockGmresSolMgr<ST,MV,OP>(rcp(&problem,false), rcp(&belosList,false)));
+
+    //
+    // **********Print out information about problem*******************
+    //
+    if (proc_verbose) {
+      std::cout << std::endl << std::endl;
+      std::cout << "Dimension of matrix: " << NumGlobalElements << std::endl;
+      std::cout << "Number of right-hand sides: " << numrhs << std::endl;
+      std::cout << "Block size used by solver: " << blocksize << std::endl;
+      std::cout << "Max number of restarts allowed: " << maxrestarts << std::endl;
+      std::cout << "Max number of Gmres iterations per linear system: " << maxiters << std::endl;
+      std::cout << "Relative residual tolerance: " << tol << std::endl;
+      std::cout << std::endl;
+    }
+    //
+    // Perform solve
+    //
+    Belos::ReturnType ret = newSolver->solve();
+    //
+    // Get the number of iterations for this solve.
+    //
+    int numIters = newSolver->getNumIters();
+    if (proc_verbose)
+      std::cout << "Number of iterations performed for this solve: " << numIters << std::endl;
+    //
+    // Compute actual residuals.
+    //
+    bool badRes = false;
+    std::vector<ST> actual_resids( numrhs );
+    std::vector<ST> rhs_norm( numrhs );
```

Comment:
These variables are snake cased which is not consistent with common variable naming in the file
I would have write `actualResids` and `rhsNorms`

Reply:
Resolved